source/codegen/templates/register_all_services.cpp.mako

```diff
@@ -24,6 +24,9 @@ repository_type_to_config = service_helpers.list_session_repository_handle_types
 %>\
 <%block filter="common_helpers.os_conditional_compile_block(config)">\
 #include "${module_name}/${module_name}_service_registrar.h"
+% if config.get("has_moniker_streaming_apis", False):
```

Comment:
You don't need this now that you are not calling `nifpga_grpc::RegisterMonikerEndpoints();` from `resgister_all_services.cpp`. And then restore/remove blank line at the end of the file so that this file doesn't appear in the review.

Reply:
Updated